python/model/flow/External.yaml

```diff
@@ -1,3 +1,2 @@
 model:
   name: User
-  behavior: []
```

Comment:
Was this meant to be removed?

Reply:
Yes, it's a hold over from when we had behaviors as a required field, but then we wanted to represent an external actor whose behavior you didn't necessarily need to model.